tests/Http/Organizer/UpdateContactPointRequestHandlerTest.php

```diff
@@ -0,0 +1,268 @@
+    {
+        $this->commandBus = new TraceableCommandBus();
+
+        $this->updateContactPointRequestHandler = new UpdateContactPointRequestHandler($this->commandBus);
+
+        $this->psr7RequestBuilder = new Psr7RequestBuilder();
+
+        $this->commandBus->record();
+    }
+
+    /**
+     * @dataProvider contactPointDataProvider
+     * @test
+     */
+    public function it_handles_updating_a_contact_point(string $body, ContactPoint $contactPoint): void
+    {
+        $updateUrlRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('organizerId', 'a088f396-ac96-45c4-b6b2-e2b6afe8af07')
+            ->withBodyFromString($body)
+            ->build('PUT');
+
+        $this->updateContactPointRequestHandler->handle($updateUrlRequest);
+
+        $this->assertEquals(
+            [
+                new UpdateContactPoint(
+                    'a088f396-ac96-45c4-b6b2-e2b6afe8af07',
+                    $contactPoint
+                ),
+            ],
+            $this->commandBus->getRecordedCommands()
+        );
+    }
+
+    public function contactPointDataProvider(): array
+    {
+        return [
+            'update contact point phone' => [
+                '{
+                    "phone": ["016 10 20 30"]
+                }',
+                new ContactPoint(
+                    new TelephoneNumbers(new TelephoneNumber('016 10 20 30')),
+                    null,
+                    null
+                ),
+            ],
+            'update contact point email' => [
+                '{
+                    "email": ["info@publiq.be"]
+                }',
+                new ContactPoint(
+                    null,
+                    new EmailAddresses(new EmailAddress('info@publiq.be')),
+                    null
+                ),
+            ],
+            'update contact point url' => [
+                '{
+                    "url": ["https://www.publiq.be"]
+                }',
+                new ContactPoint(
+                    null,
+                    null,
+                    new Urls(new Url('https://www.publiq.be'))
+                ),
+            ],
+            'update all contact point information' => [
+                '{
+                    "phone": ["016 10 20 30"],
+                    "email": ["info@publiq.be"],
+                    "url": ["https://www.publiq.be"]
+                }',
+                new ContactPoint(
+                    new TelephoneNumbers(new TelephoneNumber('016 10 20 30')),
+                    new EmailAddresses(new EmailAddress('info@publiq.be')),
+                    new Urls(new Url('https://www.publiq.be'))
+                ),
+            ],
+            'update multiple contact point information' => [
+                '{
+                    "phone": ["016 10 20 30", "016 11 22 33"],
+                    "email": ["info@publiq.be", "info@cn.be"],
+                    "url": ["https://www.publiq.be", "https://www.cn.be"]
+                }',
```

Comment:
More a general comment / FYI: I'm definitely okay with hardcoding JSON in the tests, but if it becomes too complex to write you can also use `Json::encode([... your data as an assoc array ...])` if you want

Reply:
And/or add a `withBodyFromArray()` or something like that to `Psr7RequestBuilder` that just takes an assoc array and converts it to Json so we don't have to repeat that in every test file.

Reply:
I added this new method `withBodyFromArray`.